### ISSUE TITLE
editableOptions.blurElem ignored if editableOptions.buttons == 'no'

### ISSUE THREAD

**Issue by troygilbert** [vitalets/angular-xeditable#285]
If you use `editableOptions.buttons = 'no'` in order to have all editable elements default to no buttons the logic at `dist/xeditable.js:609` will cause whatever you have for `editableOptions.blur` to be ignored and defaulted to `cancel`. I think this is more appropriate logic:

```
self.editorEl.attr('blur', self.attrs.blur || editableOptions.blurElem || 'cancel');
```

**Comment by ckosloski** [comment]
I'm not sure that is right either.  `editableOptions.blurElem` will always have a value (defaults to cancel), so the last or will never be used.

**Comment by ckosloski** [comment]
@eugef please close, this was fixed in #649
